src/util/matchEpisodeToPicture.js

```diff
@@ -0,0 +1,30 @@
+import episode1url from '../sw-covers/Star-Wars-Episode-I-The-Phantom-Menace.jpg';
```

Comment:
I think we can use an array with the paths of the cover images and based on episodeId we can return the element in the array at the position `episodeId-1`